src/test/test_db.py

```diff
@@ -41,6 +40,18 @@ def test_add_user(self):
         self.assertEqual(d.add_user(user), STATUS_OK)
         self.assertEqual(d.add_user(user), STATUS_USER_ALREADY_EXIST)
 
+    def test_deactivate_user(self):
+        d = DBDriver()
+        user_id = random.randint(10, 1e7)
+        user = {"tg_id": user_id,
+                    "first_name": "Иван",
+                    "last_name": "Иванов",
+                    "patronymic_name": "Петрович",
+                    "email": "test@ya.ru"}
+        # self.assertEqual(d.deactivate_user(user), STATUS_OK)
+        self.assertEqual(d.deactivate_user(user), STATUS_FAIL)
```

Comment:
Does it work fine? Why first attempt to deactivate the user returns STATUS_FAIL ?